test/appium/tests/atomic/chats/test_public.py

```diff
@@ -437,23 +437,73 @@ def test_different_status_in_timeline(self):
 
     @marks.testrail_id(700727)
     @marks.medium
-    def test_gap_in_public_chat(self):
+    def test_gap_in_public_chat_and_no_gap_in_1_1_and_group_chats(self):
         self.create_drivers(2)
         device_1, device_2 = SignInView(self.drivers[0]), SignInView(self.drivers[1])
         home_1, home_2 = device_1.create_user(), device_2.create_user()
-        message = "testing gap"
-        chat_name = home_1.get_random_chat_name()
+        message_1 = "testing gap"
+        message_2 = "testing no gap"
+        pub_chat_name = home_1.get_random_chat_name()
+        group_chat_name = home_1.get_random_chat_name()
+        public_key_1, username_1 = home_1.get_public_key_and_username(True)
+        public_key_2, username_2 = home_2.get_public_key_and_username(True)
         profile_1 = home_1.profile_button.click()
         profile_1.sync_settings_button.click()
         profile_1.sync_history_for_button.click()
         profile_1.element_by_translation_id("two-minutes").click()
         home_1.home_button.click()
-        chat_1, chat_2 = home_1.join_public_chat(chat_name), home_2.join_public_chat(chat_name)
-        chat_1.send_message("HI")
+        home_2.home_button.click()
```

Comment:
can be improved by list comprehension
[home.home_button.click() for home in (home_1, home_2)]